src/main/resources/assets/chattweaks/lang/de_DE.lang

```diff
@@ -0,0 +1,36 @@
+chattweaks:gui.imagePreview.openInBrowser=Im Netznavigator öffnen
+chattweaks:gui.imagePreview.copyToClipboard=In die Zwischenablage kopieren
+chattweaks:gui.imagePreview.close=Schließen
+chattweaks:gui.imagePreview.loadingPreview=Vorschau lädt...
+
+chattweaks:gui.chat.tooltipEmote=Emote:
+chattweaks:gui.chat.tooltipLocalEmotes=Lokale Emotes
+chattweaks:gui.chat.tooltipEmoteChannel=Kanal:
+chattweaks:gui.chat.tooltipBTTVEmotes=BetterTTV
+chattweaks:gui.chat.tooltipFFZEmotes=FrankerFaceZ
+chattweaks:gui.chat.tooltipTwitchEmotes=Twitch Global
+chattweaks:gui.chat.tooltipTwitchPrimeEmotes=Twitch Prime
+
+chattweaks:gui.confirm.optionalModuleRequired=Du muss einen optionalen Modul installieren für die Funktionalität von %s:
```

Comment:
Grammar is wrong and sentence structure could be better, e.g. "Die %s-Funktionalität benötigt ein..."